File: Data Structures & Algorithms/duplicate-integer/submission-4.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        if (nums.size()<2)
        {
            return false;
        }
        else 
        return unordered_set <int>(nums.begin(), nums.end()).size() < nums.size(); 
    }};

File: Data Structures & Algorithms/two-integer-sum/submission-4.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res =[]
        seen = {}
        for i,n in enumerate (nums):
            comp = target - n
            if comp in seen:
                res.append(seen[comp])
                res.append(i)
            else:
                seen[n] = i
        return res